.github/workflows/e2e.yaml

```
@@ -59,6 +58,18 @@ jobs:
           curl -sSL http://mondoo.io/download.sh | bash
           mv mondoo /usr/local/bin/ 
           make test/deployment
+      - name: Verify Mondoo Operator kustomize manifests in different namespace
+        run: |
+          kubectl delete -f config/samples/k8s_v1alpha1_mondooauditconfig.yaml
+          kubectl create ns not-mondoo-operator
+          kubectl create secret generic mondoo-client --namespace not-mondoo-operator --from-file=config=creds.json
+          yq ".metadata.namespace= \"not-mondoo-operator\" | .metadata.name = \"mondoo-operator-nodes\"" config/rbac/service_account_nodes.yaml | kubectl apply -f -
+          yq ".metadata.namespace= \"not-mondoo-operator\" | .metadata.name = \"mondoo-operator-workload\"" config/rbac/service_account_workloads.yaml  | kubectl apply -f -
+          kubectl get clusterrolebinding mondoo-operator-workload -o yaml | yq ".subjects[0].namespace=\"not-mondoo-operator-workload\"" | kubectl apply -f -
+          yq -i ".metadata.namespace= \"not-mondoo-operator\"" config/samples/k8s_v1alpha1_mondooauditconfig.yaml
+          kubectl apply -f config/samples/k8s_v1alpha1_mondooauditconfig.yaml
```

You can just do this in a single line
```suggestion
          yq '.metadata.namespace= "not-mondoo-operator"' config/samples/k8s_v1alpha1_mondooauditconfig.yaml | kubectl apply -
```